.github/workflows/release.yml

```diff
@@ -20,21 +20,24 @@ on:
 
 jobs:
   release_assurance:
-    runs-on: macos-latest
+    runs-on: macos-13
     steps:
     - uses: actions/checkout@v2
       with:
         ref: ${{ github.event.inputs.branch }}
     - uses: maxim-lobanov/setup-xcode@v1
       with:
-        xcode-version: '14.1'
+        xcode-version: '15.0.1'
```

Comment:
should this be 15.1.0 like the xcode versions we're using for our other automations?

Reply:
For release, we are using 15.0.1 in order to support the oldest version of Xcode possible for our customers.